x-pack/plugins/endpoint/server/routes/index.ts

```diff
@@ -0,0 +1,28 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License;
+ * you may not use this file except in compliance with the Elastic License.
+ */
+
+import { IRouter } from 'kibana/server';
+
+export function addRoutes(router: IRouter) {
+  router.get(
+    {
+      path: '/api/endpoint/hello-world',
+      validate: false,
+      options: {
+        tags: ['access:resolver'],
```

Comment:
just to make sure, this is for integration with the security plugin, correct?

Reply:
@andrewstucki i just made that up based on this '[plugin feature registration](https://www.elastic.co/guide/en/kibana/current/development-plugin-feature-registration.html)' webpage. i originally added this to the endpoint app PR to get some tests to pass. I'll remove it from this PR

Reply:
removed